nextflow_schema.json

```diff
@@ -607,11 +607,19 @@
                     "hidden": true,
                     "fa_icon": "fas fa-users-cog"
                 },
-                "sequencing_center": {
+                "seq_center": {
                     "type": "string",
                     "fa_icon": "fas fa-university",
-                    "description": "Name of sequencing center to be displayed in BAM file",
-                    "help_text": "It will be in the CN field",
+                    "description": "Sequencing center information to be added to read group (CN field).",
+                    "help_text": "This parameter is required for creating a proper header to use in the downstream analysis of GATK",
```

Comment:
If it is a required param, should be unhidden (?) (if that is even a word :D )

Reply:
it's not a required one as we have a sensible default

Reply:
hm ok the I find the help text misleading :D Maybe add that it defaults to ... and can be "set for be added into the header "

Reply:
yes, help text is not helpful at all, as we have no value in our tests and it works fine...

Reply:
```suggestion
```